client/src/components/UI/Backdrop/Backdrop.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from './Backdrop.module.css';
+
+const backdrop = ({ show, clicked }) => (show ? <div className={classes.Backdrop} onClick={clicked} /> : null);
```

Comment:
Is husky working properly? Eslint should send an error of line length

Reply:
Actually eslint is working on staged file and I commited and pushed this first then pulled changes from husky branch.
I am going to fix all the existing lint problems in this PR now.